docs/docsite/rst/community/collection_contributors/collection_integration_running.rst

```diff
@@ -0,0 +1,32 @@
+.. _collection_run_integration_tests:
+
+Running integration tests
+============================
+
+In the following examples, we will use ``Docker`` to run integration tests locally. Ensure you have :ref:`collection_prepare_environment` first.
```

Comment:
In the following examples, we'll use ``Docker`` to run integration tests locally. Ensure you have :ref:`collection_prepare_environment` first.

Reply:
Hi @Dule-mart and thanks for the comment. We don't use contractions (we'll) because that makes it harder for  the translation team to translate the docs.